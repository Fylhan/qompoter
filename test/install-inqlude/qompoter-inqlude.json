{
	"name": "install-ok/qompoter-inqlude",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"kde/attica": "v5.25.0",
		"cp/fylhan": "v1.1"
	},
	"require-dev": {
		"laumaya/glc-lib": "dev-master"
	}
}
