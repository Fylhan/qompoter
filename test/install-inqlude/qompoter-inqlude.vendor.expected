vendor
vendor/attica
vendor/attica/.arcconfig
vendor/attica/.reviewboardrc
vendor/attica/AUTHORS
vendor/attica/CMakeLists.txt
vendor/attica/COPYING
vendor/attica/CTestConfig.cmake
vendor/attica/CTestCustom.cmake
vendor/attica/ChangeLog
vendor/attica/KF5AtticaConfig.cmake.in
vendor/attica/README.md
vendor/attica/autotests
vendor/attica/autotests/CMakeLists.txt
vendor/attica/autotests/persontest.cpp
vendor/attica/autotests/privatedatatest.cpp
vendor/attica/metainfo.yaml
vendor/attica/src
vendor/attica/src/CMakeLists.txt
vendor/attica/src/accountbalance.cpp
vendor/attica/src/accountbalance.h
vendor/attica/src/accountbalanceparser.cpp
vendor/attica/src/accountbalanceparser.h
vendor/attica/src/achievement.cpp
vendor/attica/src/achievement.h
vendor/attica/src/achievementparser.cpp
vendor/attica/src/achievementparser.h
vendor/attica/src/activity.cpp
vendor/attica/src/activity.h
vendor/attica/src/activityparser.cpp
vendor/attica/src/activityparser.h
vendor/attica/src/atticabasejob.cpp
vendor/attica/src/atticabasejob.h
vendor/attica/src/atticautils.cpp
vendor/attica/src/atticautils.h
vendor/attica/src/buildservice.cpp
vendor/attica/src/buildservice.h
vendor/attica/src/buildservicejob.cpp
vendor/attica/src/buildservicejob.h
vendor/attica/src/buildservicejoboutput.cpp
vendor/attica/src/buildservicejoboutput.h
vendor/attica/src/buildservicejoboutputparser.cpp
vendor/attica/src/buildservicejoboutputparser.h
vendor/attica/src/buildservicejobparser.cpp
vendor/attica/src/buildservicejobparser.h
vendor/attica/src/buildserviceparser.cpp
vendor/attica/src/buildserviceparser.h
vendor/attica/src/category.cpp
vendor/attica/src/category.h
vendor/attica/src/categoryparser.cpp
vendor/attica/src/categoryparser.h
vendor/attica/src/cloud.cpp
vendor/attica/src/cloud.h
vendor/attica/src/cloudparser.cpp
vendor/attica/src/cloudparser.h
vendor/attica/src/cmake
vendor/attica/src/cmake/libKF5Attica.pc.cmake
vendor/attica/src/cmake/libattica.pc.cmake
vendor/attica/src/comment.cpp
vendor/attica/src/comment.h
vendor/attica/src/commentparser.cpp
vendor/attica/src/commentparser.h
vendor/attica/src/content.cpp
vendor/attica/src/content.h
vendor/attica/src/contentparser.cpp
vendor/attica/src/contentparser.h
vendor/attica/src/deletejob.cpp
vendor/attica/src/deletejob.h
vendor/attica/src/distribution.cpp
vendor/attica/src/distribution.h
vendor/attica/src/distributionparser.cpp
vendor/attica/src/distributionparser.h
vendor/attica/src/downloaddescription.cpp
vendor/attica/src/downloaddescription.h
vendor/attica/src/downloaditem.cpp
vendor/attica/src/downloaditem.h
vendor/attica/src/downloaditemparser.cpp
vendor/attica/src/downloaditemparser.h
vendor/attica/src/event.cpp
vendor/attica/src/event.h
vendor/attica/src/eventparser.cpp
vendor/attica/src/eventparser.h
vendor/attica/src/folder.cpp
vendor/attica/src/folder.h
vendor/attica/src/folderparser.cpp
vendor/attica/src/folderparser.h
vendor/attica/src/forum.cpp
vendor/attica/src/forum.h
vendor/attica/src/forumparser.cpp
vendor/attica/src/forumparser.h
vendor/attica/src/getjob.cpp
vendor/attica/src/getjob.h
vendor/attica/src/homepageentry.cpp
vendor/attica/src/homepageentry.h
vendor/attica/src/homepagetype.cpp
vendor/attica/src/homepagetype.h
vendor/attica/src/homepagetypeparser.cpp
vendor/attica/src/homepagetypeparser.h
vendor/attica/src/icon.cpp
vendor/attica/src/icon.h
vendor/attica/src/itemjob.cpp
vendor/attica/src/itemjob.h
vendor/attica/src/knowledgebaseentry.cpp
vendor/attica/src/knowledgebaseentry.h
vendor/attica/src/knowledgebaseentryparser.cpp
vendor/attica/src/knowledgebaseentryparser.h
vendor/attica/src/license.cpp
vendor/attica/src/license.h
vendor/attica/src/licenseparser.cpp
vendor/attica/src/licenseparser.h
vendor/attica/src/listjob.cpp
vendor/attica/src/listjob.h
vendor/attica/src/listjob_inst.cpp
vendor/attica/src/message.cpp
vendor/attica/src/message.h
vendor/attica/src/messageparser.cpp
vendor/attica/src/messageparser.h
vendor/attica/src/metadata.cpp
vendor/attica/src/metadata.h
vendor/attica/src/parser.cpp
vendor/attica/src/parser.h
vendor/attica/src/person.cpp
vendor/attica/src/person.h
vendor/attica/src/personparser.cpp
vendor/attica/src/personparser.h
vendor/attica/src/platformdependent.h
vendor/attica/src/platformdependent_v2.cpp
vendor/attica/src/platformdependent_v2.h
vendor/attica/src/postfiledata.cpp
vendor/attica/src/postfiledata.h
vendor/attica/src/postjob.cpp
vendor/attica/src/postjob.h
vendor/attica/src/privatedata.cpp
vendor/attica/src/privatedata.h
vendor/attica/src/privatedataparser.cpp
vendor/attica/src/privatedataparser.h
vendor/attica/src/project.cpp
vendor/attica/src/project.h
vendor/attica/src/projectparser.cpp
vendor/attica/src/projectparser.h
vendor/attica/src/provider.cpp
vendor/attica/src/provider.h
vendor/attica/src/providermanager.cpp
vendor/attica/src/providermanager.h
vendor/attica/src/publisher.cpp
vendor/attica/src/publisher.h
vendor/attica/src/publisherfield.cpp
vendor/attica/src/publisherfield.h
vendor/attica/src/publisherfieldparser.cpp
vendor/attica/src/publisherfieldparser.h
vendor/attica/src/publisherparser.cpp
vendor/attica/src/publisherparser.h
vendor/attica/src/putjob.cpp
vendor/attica/src/putjob.h
vendor/attica/src/qtplatformdependent.cpp
vendor/attica/src/qtplatformdependent_p.h
vendor/attica/src/remoteaccount.cpp
vendor/attica/src/remoteaccount.h
vendor/attica/src/remoteaccountparser.cpp
vendor/attica/src/remoteaccountparser.h
vendor/attica/src/topic.cpp
vendor/attica/src/topic.h
vendor/attica/src/topicparser.cpp
vendor/attica/src/topicparser.h
vendor/attica/src/version.h.cmake
vendor/attica/tests
vendor/attica/tests/CMakeLists.txt
vendor/attica/tests/projecttest
vendor/attica/tests/projecttest/CMakeLists.txt
vendor/attica/tests/projecttest/editproject.ui
vendor/attica/tests/projecttest/main.cpp
vendor/attica/tests/projecttest/projecttest-0.1.kdev4
vendor/attica/tests/projecttest/projecttest.cpp
vendor/attica/tests/projecttest/projecttest.h
vendor/fylhan
vendor/fylhan/qompoter.json
vendor/fylhan/qompoter.pri
vendor/glc-lib
vendor/glc-lib/LICENSE.LGPL
vendor/glc-lib/README
vendor/glc-lib/doc
vendor/glc-lib/doc/GLC_lib-Doxygen-Doc
vendor/glc-lib/doc/glc_footer.html
vendor/glc-lib/glc_lib.pri
vendor/glc-lib/glc_lib.pro
vendor/glc-lib/install.pri
vendor/glc-lib/sonar-project.properties
vendor/glc-lib/src
vendor/glc-lib/src/examples
vendor/glc-lib/src/examples/example01
vendor/glc-lib/src/examples/example02
vendor/glc-lib/src/examples/example03
vendor/glc-lib/src/examples/example04
vendor/glc-lib/src/examples/example05
vendor/glc-lib/src/examples/example06
vendor/glc-lib/src/examples/example07
vendor/glc-lib/src/examples/example08
vendor/glc-lib/src/examples/example09
vendor/glc-lib/src/examples/example10
vendor/glc-lib/src/examples/example11
vendor/glc-lib/src/examples/examples.pro
vendor/glc-lib/src/lib
vendor/glc-lib/src/lib/3DWidget
vendor/glc-lib/src/lib/3rdparty
vendor/glc-lib/src/lib/GLC_3DRep
vendor/glc-lib/src/lib/GLC_3DViewCollection
vendor/glc-lib/src/lib/GLC_3DViewInstance
vendor/glc-lib/src/lib/GLC_3DWidget
vendor/glc-lib/src/lib/GLC_3DWidgetManager
vendor/glc-lib/src/lib/GLC_3DWidgetManagerHandle
vendor/glc-lib/src/lib/GLC_AbstractManipulator
vendor/glc-lib/src/lib/GLC_Arrow
vendor/glc-lib/src/lib/GLC_Attributes
vendor/glc-lib/src/lib/GLC_Axis
vendor/glc-lib/src/lib/GLC_BSRep
vendor/glc-lib/src/lib/GLC_BoundingBox
vendor/glc-lib/src/lib/GLC_Box
vendor/glc-lib/src/lib/GLC_CacheManager
vendor/glc-lib/src/lib/GLC_Camera
vendor/glc-lib/src/lib/GLC_Circle
vendor/glc-lib/src/lib/GLC_Cone
vendor/glc-lib/src/lib/GLC_Context
vendor/glc-lib/src/lib/GLC_ContextManager
vendor/glc-lib/src/lib/GLC_CuttingPlane
vendor/glc-lib/src/lib/GLC_Cylinder
vendor/glc-lib/src/lib/GLC_Disc
vendor/glc-lib/src/lib/GLC_ErrorLog
vendor/glc-lib/src/lib/GLC_Exception
vendor/glc-lib/src/lib/GLC_Ext
vendor/glc-lib/src/lib/GLC_ExtrudedMesh
vendor/glc-lib/src/lib/GLC_Factory
vendor/glc-lib/src/lib/GLC_FileFormatException
vendor/glc-lib/src/lib/GLC_FileLoader
vendor/glc-lib/src/lib/GLC_FlyMover
vendor/glc-lib/src/lib/GLC_Frustum
vendor/glc-lib/src/lib/GLC_GeomTools
vendor/glc-lib/src/lib/GLC_Geometry
vendor/glc-lib/src/lib/GLC_Global
vendor/glc-lib/src/lib/GLC_Glu
vendor/glc-lib/src/lib/GLC_ImagePlane
vendor/glc-lib/src/lib/GLC_InputEventInterpreter
vendor/glc-lib/src/lib/GLC_Interpolator
vendor/glc-lib/src/lib/GLC_Light
vendor/glc-lib/src/lib/GLC_Line
vendor/glc-lib/src/lib/GLC_Line3d
vendor/glc-lib/src/lib/GLC_Log
vendor/glc-lib/src/lib/GLC_Material
vendor/glc-lib/src/lib/GLC_Matrix4x4
vendor/glc-lib/src/lib/GLC_Mesh
vendor/glc-lib/src/lib/GLC_Mover
vendor/glc-lib/src/lib/GLC_MoverController
vendor/glc-lib/src/lib/GLC_Object
vendor/glc-lib/src/lib/GLC_Octree
vendor/glc-lib/src/lib/GLC_OctreeNode
vendor/glc-lib/src/lib/GLC_OpenGlException
vendor/glc-lib/src/lib/GLC_PanMover
vendor/glc-lib/src/lib/GLC_Plane
vendor/glc-lib/src/lib/GLC_Point
vendor/glc-lib/src/lib/GLC_Point2d
vendor/glc-lib/src/lib/GLC_Point2df
vendor/glc-lib/src/lib/GLC_Point3d
vendor/glc-lib/src/lib/GLC_Point3df
vendor/glc-lib/src/lib/GLC_Point4d
vendor/glc-lib/src/lib/GLC_PointCloud
vendor/glc-lib/src/lib/GLC_PointSprite
vendor/glc-lib/src/lib/GLC_Polylines
vendor/glc-lib/src/lib/GLC_PullManipulator
vendor/glc-lib/src/lib/GLC_QuickCamera
vendor/glc-lib/src/lib/GLC_QuickItem
vendor/glc-lib/src/lib/GLC_QuickOccurrence
vendor/glc-lib/src/lib/GLC_QuickSelection
vendor/glc-lib/src/lib/GLC_QuickView
vendor/glc-lib/src/lib/GLC_Rectangle
vendor/glc-lib/src/lib/GLC_RenderProperties
vendor/glc-lib/src/lib/GLC_RenderState
vendor/glc-lib/src/lib/GLC_RenderStatistics
vendor/glc-lib/src/lib/GLC_Renderer
vendor/glc-lib/src/lib/GLC_Rep
vendor/glc-lib/src/lib/GLC_RepCrossMover
vendor/glc-lib/src/lib/GLC_RepFlyMover
vendor/glc-lib/src/lib/GLC_RepMover
vendor/glc-lib/src/lib/GLC_RepTrackBallMover
vendor/glc-lib/src/lib/GLC_RotationManipulator
vendor/glc-lib/src/lib/GLC_ScreenShotSettings
vendor/glc-lib/src/lib/GLC_SelectionEvent
vendor/glc-lib/src/lib/GLC_SelectionMaterial
vendor/glc-lib/src/lib/GLC_SelectionSet
vendor/glc-lib/src/lib/GLC_Shader
vendor/glc-lib/src/lib/GLC_SpacePartitioning
vendor/glc-lib/src/lib/GLC_Sphere
vendor/glc-lib/src/lib/GLC_State
vendor/glc-lib/src/lib/GLC_StructInstance
vendor/glc-lib/src/lib/GLC_StructOccurrence
vendor/glc-lib/src/lib/GLC_StructReference
vendor/glc-lib/src/lib/GLC_Texture
vendor/glc-lib/src/lib/GLC_TraceLog
vendor/glc-lib/src/lib/GLC_TrackBallMover
vendor/glc-lib/src/lib/GLC_TsrMover
vendor/glc-lib/src/lib/GLC_TurnTableMover
vendor/glc-lib/src/lib/GLC_UserInput
vendor/glc-lib/src/lib/GLC_Vector2d
vendor/glc-lib/src/lib/GLC_Vector2df
vendor/glc-lib/src/lib/GLC_Vector3d
vendor/glc-lib/src/lib/GLC_Vector3df
vendor/glc-lib/src/lib/GLC_Vector4d
vendor/glc-lib/src/lib/GLC_ViewHandler
vendor/glc-lib/src/lib/GLC_Viewport
vendor/glc-lib/src/lib/GLC_World
vendor/glc-lib/src/lib/GLC_WorldReaderHandler
vendor/glc-lib/src/lib/GLC_WorldReaderPlugin
vendor/glc-lib/src/lib/GLC_WorldTo3ds
vendor/glc-lib/src/lib/GLC_WorldTo3dxml
vendor/glc-lib/src/lib/GLC_ZoomMover
vendor/glc-lib/src/lib/geometry
vendor/glc-lib/src/lib/glcXmlUtil
vendor/glc-lib/src/lib/glc_boundingbox.cpp
vendor/glc-lib/src/lib/glc_boundingbox.h
vendor/glc-lib/src/lib/glc_cachemanager.cpp
vendor/glc-lib/src/lib/glc_cachemanager.h
vendor/glc-lib/src/lib/glc_config.h
vendor/glc-lib/src/lib/glc_context.cpp
vendor/glc-lib/src/lib/glc_context.h
vendor/glc-lib/src/lib/glc_contextmanager.cpp
vendor/glc-lib/src/lib/glc_contextmanager.h
vendor/glc-lib/src/lib/glc_contextshareddata.cpp
vendor/glc-lib/src/lib/glc_contextshareddata.h
vendor/glc-lib/src/lib/glc_errorlog.cpp
vendor/glc-lib/src/lib/glc_errorlog.h
vendor/glc-lib/src/lib/glc_exception.cpp
vendor/glc-lib/src/lib/glc_exception.h
vendor/glc-lib/src/lib/glc_ext.cpp
vendor/glc-lib/src/lib/glc_ext.h
vendor/glc-lib/src/lib/glc_factory.cpp
vendor/glc-lib/src/lib/glc_factory.h
vendor/glc-lib/src/lib/glc_fileformatexception.cpp
vendor/glc-lib/src/lib/glc_fileformatexception.h
vendor/glc-lib/src/lib/glc_global.cpp
vendor/glc-lib/src/lib/glc_global.h
vendor/glc-lib/src/lib/glc_lib.qrc
vendor/glc-lib/src/lib/glc_log.cpp
vendor/glc-lib/src/lib/glc_log.h
vendor/glc-lib/src/lib/glc_object.cpp
vendor/glc-lib/src/lib/glc_object.h
vendor/glc-lib/src/lib/glc_openglexception.cpp
vendor/glc-lib/src/lib/glc_openglexception.h
vendor/glc-lib/src/lib/glc_renderstatistics.cpp
vendor/glc-lib/src/lib/glc_renderstatistics.h
vendor/glc-lib/src/lib/glc_selectionevent.cpp
vendor/glc-lib/src/lib/glc_selectionevent.h
vendor/glc-lib/src/lib/glc_state.cpp
vendor/glc-lib/src/lib/glc_state.h
vendor/glc-lib/src/lib/glc_tracelog.cpp
vendor/glc-lib/src/lib/glc_tracelog.h
vendor/glc-lib/src/lib/glc_uniformshaderdata.cpp
vendor/glc-lib/src/lib/glc_uniformshaderdata.h
vendor/glc-lib/src/lib/glu
vendor/glc-lib/src/lib/io
vendor/glc-lib/src/lib/lib.pro
vendor/glc-lib/src/lib/maths
vendor/glc-lib/src/lib/qml
vendor/glc-lib/src/lib/sceneGraph
vendor/glc-lib/src/lib/shading
vendor/glc-lib/src/lib/viewport
vendor/glc-lib/src/plugins
vendor/glc-lib/src/plugins/GLC_ViewerItem
vendor/glc-lib/src/plugins/plugins.pro
vendor/qompote.pri
vendor/vendor.pri
